src/convertToObject.js

```diff
@@ -12,7 +12,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const formatString = sourceString
+    .split(';')
+    .map(property => property.trim().split(':'))
+    .filter(property => property.length === 2);
+
+  const cssObj = {};
+
+  for (const key of formatString) {
```

Comment:
can use ForEach

Reply:
Yep, we need to learn how to use array methods instead of loops. Rewrite using `forEach`